d2common/d2resource/resource_paths.go

```diff
@@ -110,7 +110,7 @@ const (
 	HealthManaIndicator = "/data/global/ui/PANEL/hlthmana.DC6"
 	AddSkillButton      = "/data/global/ui/PANEL/level.DC6"
 	GoldMoveDialog      = "/data/global/ui/menu/dialogbackground.DC6"
-	WPTabs              = "/data/global/ui/menu/expwaygatetabs.dc6"
+	WPTabs              = "/data/global/ui/MENU/expwaygatetabs.dc6"
```

Comment:
did this capitalization prevent the file from loading?

Reply:
right, it doesn't matter